src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -1976,8 +1982,8 @@ public synchronized void run() {
         private final Executor mExecutor = Executors.newSingleThreadExecutor();
     }
 
-    private class UnsupportedUpdatesFromMixpanel implements UpdatesFromMixpanel {
-        public UnsupportedUpdatesFromMixpanel(Tweaks tweaks) {
+    /* package */ class NoOpUpdatesFromMixpanel implements UpdatesFromMixpanel {
```

Comment:
I'm not exactly sure why you change the name of this function. First of all, I think the old name does indicate what's going on here. Second, we have another function called `UnsupportedUpdatesListener`. I'd like to keep the same naming approach so that people won't have to second guess what this function does.